### PR TITLE
mgr: drop repeated log info. and unnecessary write permission

### PR DESCRIPTION
**Pull request by Yan-waller**
In the front of this file, `__func__` has been added to dout_prefix as below:
` #define dout_prefix *_dout << "mgr.server " << __func__ << " "`

### CODE REVIEW

File: src/mgr/DaemonServer.cc [hunk 1]
```diff
@@ -176,7 +176,7 @@ bool DaemonServer::ms_verify_authorizer(Connection *con,
     if (peer_type == CEPH_ENTITY_TYPE_OSD) {
       Mutex::Locker l(lock);
       s->osd_id = atoi(s->entity_name.get_id().c_str());
-      dout(10) << __func__ << " registering osd." << s->osd_id << " session "
+      dout(10) << " registering osd." << s->osd_id << " session "
```
Comment (Liuchang0812) [inline src/mgr/DaemonServer.cc]: NIT, ` #define dout_prefix *_dout << "mgr.server " << __func__ << " "`, do not need the start space.

File: src/mgr/DaemonServer.cc [hunk 2]
```diff
@@ -214,7 +214,7 @@ bool DaemonServer::ms_handle_reset(Connection *con)
     }
     session->put(); // SessionRef takes a ref
     Mutex::Locker l(lock);
-    dout(10) << __func__ << " unregistering osd." << session->osd_id
+    dout(10) << " unregistering osd." << session->osd_id
```
Comment (Liuchang0812) [inline src/mgr/DaemonServer.cc]: ditto

**Comment by Yan-waller** [comment]
@Liuchang0812 fixed it and thanks.

File: src/mgr/DaemonServer.cc [hunk 3]
```diff
@@ -451,7 +451,7 @@ bool DaemonServer::handle_command(MCommand *m)
         con->mark_disposable();
       }
 
-      dout(1) << "do_command r=" << r << " " << rs << dendl;
+      dout(1) << "do_command " << cpp_strerror(r) << " " << rs << dendl;
```
Comment (tchaikov) [inline src/mgr/DaemonServer.cc]: would be better if we can s/do_command/handle_command/ .

Comment (Yan-waller) [inline src/mgr/DaemonServer.cc]: @tchaikov fixed it,  thanks.

File: src/mgr/DaemonServer.h [hunk 4]
```diff
@@ -21,6 +21,7 @@
 
 #include "common/Mutex.h"
 #include "common/LogClient.h"
+#include "common/errno.h"
```
Comment (tchaikov) [inline src/mgr/DaemonServer.h]: why would you include this file in the header file, but to use `cpp_strerror()` in `.cc`?

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-27_05:44:05-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**